corehq/apps/export/static/export/js/create_export.js

```diff
@@ -141,6 +141,14 @@ hqDefine("export/js/create_export", [
                     $('#div_id_' + fieldSlug).find("label").text(self._labels[fieldSlug]); 
                     var $formElem = $('#id_' + fieldSlug);
                     if ($formElem.length > 0) {
+                        if ($formElem.hasClass("select2-hidden-accessible")) {
```

Comment:
I was going to complain about this being an obscure way to figure out if the select2 has already been initialized, then looked it up and realized this is the documented way to do it.